sdv/multi_table/base.py

```diff
@@ -71,7 +71,10 @@ def _print(self, text='', **kwargs):
 
     def __init__(self, metadata, locales=None, synthesizer_kwargs=None):
         self.metadata = metadata
-        self.metadata.validate()
+        with warnings.catch_warnings():
+            warnings.filterwarnings('ignore', message=r'.*column relationship.*')
+            self.metadata.validate()
```

Comment:
where is the other warning being raised?

Reply:
It's raised here:
https://github.com/sdv-dev/SDV/blob/21b064a75f06f6f68997110b7e2260fb73026341/sdv/multi_table/base.py#L93

When we defines the single table synthesizers:
https://github.com/sdv-dev/SDV/blob/21b064a75f06f6f68997110b7e2260fb73026341/sdv/multi_table/base.py#L55

Reply:
Got it. Maybe in the future we should have a way to skip the validation if it has already been called from the multi-table level